Project.toml

```diff
@@ -40,7 +40,28 @@ Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 WriteVTK = "64499a7a-5c06-52f2-abe2-ccb03c286192"
 
 [compat]
+Combinatorics = "1.0.0"
```

Comment:
We probably should use tilde operators:
https://julialang.github.io/Pkg.jl/v1/compatibility/index.html#Tilde-specifiers-1

Reply:
@vchuravy is that correct?

Reply:
According to [some General docs](https://github.com/JuliaRegistries/General/blob/master/README.md#automatic-merging-of-pull-requests), it seems that we want to include upper bounds, but using the tilde seems to not provide upper bounds. It's not clear to me which to follow.

Reply:
Yes.If the patch version is significant otherwise I would use just `1.0`